pyp5js/tests/http/test_web_app.py

```diff
@@ -0,0 +1,108 @@
+import os
+import shutil
+from pyp5js.config import SKETCHBOOK_DIR
+from pyp5js.http.web_app import app as web_app
+from flask_testing import TestCase
+
+class Pyp5jsWebTestCase(TestCase):
+    render_templates = False
+
+    def create_app(self):
+        app = web_app
+        app.config['TESTING'] = True
+        return app
+
+    def setUp(self):
+        if not SKETCHBOOK_DIR.exists():
+            SKETCHBOOK_DIR.mkdir()
+
+    def tearDown(self):
+        if SKETCHBOOK_DIR.exists():
+            shutil.rmtree(SKETCHBOOK_DIR)
+
+    def create_sketch(self, name):
+        sketch_dir = SKETCHBOOK_DIR.joinpath(name)
+        os.makedirs(sketch_dir)
+        target_file = sketch_dir.joinpath(f'{name}.py')
+        index_file = sketch_dir.joinpath('index.html')
+        self.create_file(target_file)
+        self.create_file(index_file, 'index file content')
+        return sketch_dir
```

Comment:
A refactoring suggestion here is to use the `commands` API. It already handles all the required files and directory creation. So it's highly possible that you can refactor this to:

```python
from pyp5js import commands

def create_sketch(self, name):
    sketch_files = commands.new(name)
    return sketch_files.sketch_dir
```